docs/mine-hnt/validators/troubleshooting.mdx

```diff
@@ -20,24 +20,56 @@ This can be run directly on the Docker container as well:
 
 ```docker exec validator tail -f /var/data/log/console.log```
 
-
 ## Compromised server
 If you find that the server has been compromised just wipe it (save the blocks to avoid having to resync the entire blockchain), close the hole, stand up a new validator and then transfer the stake to it.
 
-
 ## Syncing to the blockchain
 Syncing to the blockchain could be impacted for the following reasons:
 * slow internet connection
 * your validator is disconnecting
 * you just launched an AMI instance (if using AWS)
 
+## Grab the latest blessed snapshot from Helium
+* On a running Validator run `miner snapshot list`
+* Find the highest "Height" block and download the snapshot from Helium via `curl -O https://snapshots.helium.wtf/mainnet/snap-<height>`
+* Move that file into your `validator_data` directory
+* Temporarily pause transaction sync via ```miner repair sync_pause```
+* Cancel any in-progress transaction sync via ```miner repair sync_cancel```
+* Load snapshot from downloaded file ```miner snapshot load /var/data/snap-<height>```
+* Resume transaction syncing via ```miner repair sync_resume```
+* Review `miner info p2p_status` and confirm that your validator height is >= `<height>` from what you found in the first step
+* Wait at least 5 minutes and re-confirm that your chain is adding more blocks
+
+NOTE: If you build the Validator from source the paths may be different
+
+## Grab a snapshot from another Validator
+
+:::info
+
+Besure to trust the Validator that you take your snapshot from, shenanigans could ensue! This an advanced procedure use with caution!
+
+:::
+
+* On the source Validator run `miner info p2p_status`
+* Temporarily pause transaction sync via ```miner repair sync_pause```
+* Cancel any in-progress transaction sync via ```miner repair sync_cancel```
+* On the source Validator take a snapshot via `miner snapshot take /var/data/snap-<height>` using the height you determined from the first step. 
+* On the source Validator resume transaction syncing via ```miner repair sync_resume```
+* Move the `/var/data/snap-<height>` file into your root of the `validator_data` directory on the destination Validator
```

Comment:
For docker builds, `/var/data/...` is inside the container, it will be `validator_data` on the host of the source validator.